src/ansys/fluent/core/session.py

```diff
@@ -103,15 +106,24 @@ def __init__(self, fluent_connection: FluentConnection):
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
         """
-        BaseSession.build_from_fluent_connection(self, fluent_connection)
+        remote_file_handler = RemoteFileHandler(
+            transfer_service=PimFileTransferService(fluent_connection._remote_instance)
+        )
+        BaseSession.build_from_fluent_connection(
+            self, fluent_connection, remote_file_handler
+        )
```

Comment:
This is mysterious

Reply:
@seanpearsonuk 

I thought adding it here will avoid following duplication in 2 files. 

`session_pure_meshing.py` and `session_solver.py`

![image](https://github.com/ansys/pyfluent/assets/106588300/7f93fd0a-3732-473d-b519-67e30242d2be)

`----------------------------------------------`

![image](https://github.com/ansys/pyfluent/assets/106588300/fd9e6699-a1ae-4362-8e9c-93e0dc55256e)

Reply:
I agree that it's good to avoid duplication. I was just trying to understand the particular design changes here.